src/promptflow/promptflow/_sdk/entities/_chat_group/_chat_role.py

```diff
@@ -95,6 +95,24 @@ def _build_role_io(self, flow: Union[str, PathLike], inputs_value: Dict = None):
             )
         return ChatRoleInputs(inputs), ChatRoleOutputs(outputs)
 
+    def _update_inputs_from_data_and_inputs(self, data: Dict, inputs: Dict):
+        """Update inputs from data and inputs from experiment"""
+        data_prefix = "${data."
+        inputs_prefix = "${inputs."
+        for key in self._inputs:
+            current_input = self._inputs[key]
+            value = current_input["value"]
+            if isinstance(value, str):
```

Comment:
Do we need to differentiate between case and lowercase? For example: **"${Data."**

Reply:
All binding values are lowercase, we don't expect to handle user mis-spell here. If it's not resolved here due to case issue, it will raise error when invoking chat group.